_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
+- Pan-Canadian Trust Framework
+- Some benefits integrated with 3rd parties
+
 ## Businesses
 
+- Able to find and integrate with 50% of ESDC services
+
 ## Non-profits
 
-## IITB
+- Able to find and integrate with 50% of ESDC services
+
+## IITB-ESDC
+
+- IITB is now involved in business discussions and decisions
+- 75% of employees are equipped with mobile devices allowing them to work from anywhere
+- Remote employees can always join a meeting or discussion - Digital First
+- Employees are regularly completing online courses and attending conferences related to technology and business
+- Employees are sharing their experiences and best practices (Tech Talks)
+- All new code is developed in the open by default
+- All new projects are API first
+- All new projects use cloud infrastructure
+- Enterprise Architecture (EA) actively considers open source software and open standards first
+- Teams work using Agile and DevOps methodologies
+- Continuous Integration (CI), Continuous Delivery (CD) and automation are the norm
 
 ## ESDC Programs
 
```

Comment:
In the next 5 years some of the things that may be more common - Use of data-analytics to deliver efficient programs

And for Government partners
- Digital 9 
- Citizen engagement?
- Anti-trust partnership? (Though doesn`t seem to the scope of IITB, but not sure if it would be for ESDC)

Reply:
How would this impact or change the way services are delivered?